plugins/modules/win_certificate_store.ps1

```diff
@@ -262,10 +275,7 @@ Function New-CertFile($module, $cert, $path, $type, $password) {
         $cert_bytes = $file_encoding.GetBytes($cert_content)
     }
     elseif ($type -eq "pkcs12") {
-        $module.Result.key_exported = $false
-        if ($null -ne $cert.PrivateKey) {
-            $module.Result.key_exportable = $cert.PrivateKey.CspKeyContainerInfo.Exportable
-        }
+        $module.Result.key_exportable = [string]$certPrivateKey.ExportPolicy
```

Comment:
This is changing the value around from being a bool to a string value. If you are wanting to add the export policy value as a return value we would need to add it as a new return value to ensure backwards compatibility.